features/features.go

```diff
@@ -354,12 +354,12 @@ type Checklist struct {
 
 	// TODO: Check on this.
 	Observability struct {
-		Used             bool // PROGRESS: DONE - Don't think this can be detected
-		OpenTelemetry    bool // PROGRESS: DONE - Don't think this can be detected
+		Used             bool // PROGRESS: DONE
+		OpenTelemetry    bool // PROGRESS: DONE
 		MetricsURL       bool // PROGRESS: DONE - Don't think this can be detected
 		TracePropogation bool // PROGRESS: DONE - Don't think this can be detected
-		Headers          bool // PROGRESS: DONE - Don't think this can be detected
-		Attrubutes       bool // PROGRESS: DONE - Don't think this can be detected
+		Headers          bool // PROGRESS: DONE
+		Attrubutes       bool // PROGRESS: DONE - Is this "Resource Attributes"?
```

Comment:
Yes, it is resource_attributes, I'm pretty sure. Setting "Attributes" in the Console UI sets that JSON in the metadata.

Reply:
Cool I've added a check for this.